File: 3227-vowels-game-in-a-string/3227-vowels-game-in-a-string.cpp
class Solution {
 public:
  bool doesAliceWin(string s) {
    return ranges::any_of(s, [=](char c) { return isVowel(c); });
  }
 private:
  bool isVowel(char c) {
    static constexpr string_view kVowels = "aeiou";
    return kVowels.find(c) != string_view::npos;
  }
};

File: 3227-vowels-game-in-a-string/3227-vowels-game-in-a-string.java
class Solution {
  public boolean doesAliceWin(String s) {
    return s.chars().anyMatch(c -> isVowel((char) c));
  }
  private boolean isVowel(char c) {
    return "aeiou".indexOf(Character.toLowerCase(c)) != -1;
  }
}